_toc.yml

```diff
@@ -1,45 +1,80 @@
 - file: index
 - chapters:
-  - file: community/index
+  - file: tutorials/start_index
     sections:
-    - file: community/mission_and_values
-    - file: community/team
-    - file: community/code_of_conduct
-    - file: community/code_of_conduct_reporting
-    - file: community/governance
-    - file: community/working_groups
-    title: Community
+    - file: tutorials/fundamentals/installation
+    - file: tutorials/fundamentals/getting_started
+    - file: plugins/stable/find-and-install-plugin
+    - file: tutorials/fundamentals/viewer
+    - file: tutorials/fundamentals/image
+    - file: tutorials/fundamentals/labels
+    - file: tutorials/fundamentals/points
+    - file: tutorials/fundamentals/shapes
+    - file: tutorials/fundamentals/surface
+    - file: tutorials/fundamentals/tracks
+    - file: tutorials/fundamentals/vectors
+    title: Getting Started
```

Comment:
I know we had discussed putting the title in the beginning of the section, but because this is a yaml dump I thought it could be either ignored or left to another PR.

Reply:
Ok, no worries